lib/search/mapping.rb

```diff
@@ -0,0 +1,11 @@
+module Search
+  class Mapping
```

Comment:
Would it be good to inline this into `entity_expander.rb` to indicate that it's only used in that context? Otherwise, an comment for this class would make it clear to the reader what the purpose is.